docs/en/operation/Admin-CLI.md

```diff
@@ -49,16 +49,41 @@ Back up to a specific directory on the leading master's local filesystem.
 Successfully backed up journal to /opt/alluxio/backups/alluxio-backup-2018-5-29-1527644810.gz on master Master2
 ```
 
-### checkpoint 
+### journal
```

Comment:
So many levels, will it be better to arrange like

### journal 

The `journal` command provides several sub-commands for journal management.
It has two sub commands:
* `quorum` will query and manage embedded journal powered leader election.
```console
# get information on existing state of the `MASTER` or `JOB_MASTER` leader election quorum
$ ./bin/alluxio fsadmin journal quorum info -domain <MASTER | JOB_MASTER>

# remove a member from leader election quorum
$ ./bin/alluxio fsadmin journal quorum remove -domain <MASTER | JOB_MASTER> -address <Member_Address>
```

* `checkpoint` will creates a checkpoint in the primary master journal system.
This command is mainly used for debugging and to avoid master journal logs from growing unbounded.
Checkpointing requires a pause in master metadata changes, so use this command sparingly to avoid interfering with other users of the system.
```console
$ ./bin/alluxio fsadmin journal checkpoint
```